letter-shop/.eslintignore

```diff
@@ -0,0 +1 @@
+/completed
```

Comment:
why?

Reply:
The folder has it's own eslint config. 

If you want to run the tests inside the `completed` folder then you can do that and it will use it's own config.

Note the config is different, I relax one of the rules for the parent as the code is full of unused variables because it is incomplete.

Reply:
Ah, right. I think the .eslintrc in the subfolder would override even if you don't .eslintignore it in the parent, so if you remove this .eslintignore, it would apply all the correct rules in the different folders if you run eslint in the root

Reply:
I'm using rules defined in `package.json` (note: no .eslintrc, the eslint website says this is a deprecated way of defining rules) and it didn't seem to pick those up in the subfolders.